src/app/app.component.ts

```diff
@@ -636,6 +644,39 @@ export class AppComponent implements OnInit {
     return this.calcStatusData(sales, dur);
   }
 
+  getPieTempData() {
+    if (this.tempXValueForComboPie){
+      let data = [];
+      this.dateDataWithOrWithoutRange.forEach(item => {
+        const dataPoint = item.series.find(data => data.name === this.tempXValueForComboPie);
+        data.push({ name: item.name,
+                    value: dataPoint.value,
+                    extra: {
+                      code: item.name.toLowerCase()
+                    }});
+      })
+      return data;
+    } else {
+      let sumResult = []
+      this.dateDataWithOrWithoutRange.forEach(countryData => {
+        const countryName = countryData.name;
+        let sum = 0;
+        countryData.series.forEach(seriesItem => {
+            sum += seriesItem.value;
```

Comment:
Indent once instead of twice to match coding style.
```suggestion
          sum += seriesItem.value;
```